components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <article class="bx--grid page-section qiskit-capabilities-section">
```

Comment:
Related to the comment in lines 17 to 22.

```suggestion
  <article class="bx--grid page-section">
```